### PR TITLE
Closes #365 | Implement dataloader for KDE4

### LINKED ISSUE #365: Create dataset loader for KDE4
Dataloader name: `kde4/kde4.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?kde4

|     Dataset| kde4 |
|-------------|---|
| Description | A parallel corpus of KDE4 localization files. The corpus is available in 92 languages in total, with 4099 bitexts. |
| Subsets | - |
| Languages | ind, khm, zlm, tha, vie |
| Tasks | Machine Translation |
| License | Unknown (unknown) |
| Homepage | https://opus.nlpl.eu/KDE4.php |
| HF URL | - |
| Paper URL | http://www.lrec-conf.org/proceedings/lrec2012/pdf/463_Paper.pdf |

### PR DESCRIPTION
**Pull request by ssun32**
Closes #365 

### Checkbox
- [ ] Confirm that this PR is linked to the dataset issue.
- [ ] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [ ] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [ ] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [ ] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [ ] Confirm dataloader script works with `datasets.load_dataset` function.
- [ ] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/kde4/kde4.py
Comment (holylovenia) [inline seacrowd/sea_datasets/kde4/kde4.py:59]: Hi @ssun32, would it be possible to use the languages' 3-letter ISO codes instead of these 2-letter ids for ease of reference?

**Comment by yongzx** [comment]
Tested this with `python -m tests.test_seacrowd seacrowd/sea_datasets/kde4/kde4.py` and encountered the following error. @ssun32 did you download them locally when you create this, or is it that perhaps the url currently down? 
```
File "/Users/yong/Dev/env_seacrowd/lib/python3.8/site-packages/datasets/utils/file_utils.py", line 596, in get_from_cache
    raise FileNotFoundError(f"Couldn't find file at {url}")
FileNotFoundError: Couldn't find file at https://opus.nlpl.eu/download.php?f=KDE4/v2/moses/af-ms.txt.zip
```

File: seacrowd/sea_datasets/kde4/kde4.py
Comment (holylovenia) [reply seacrowd/sea_datasets/kde4/kde4.py:59]: Also, please add a `__init__.py` file.

Comment (ssun32) [reply seacrowd/sea_datasets/kde4/kde4.py:253]: @holylovenia Converted everything to iso639-3 and added __init__.py

**Comment by ssun32** [comment]
> Tested this with `python -m tests.test_seacrowd seacrowd/sea_datasets/kde4/kde4.py` and encountered the following error. @ssun32 did you download them locally when you create this, or is it that perhaps the url currently down?
> 
> ```
> File "/Users/yong/Dev/env_seacrowd/lib/python3.8/site-packages/datasets/utils/file_utils.py", line 596, in get_from_cache
>     raise FileNotFoundError(f"Couldn't find file at {url}")
> FileNotFoundError: Couldn't find file at https://opus.nlpl.eu/download.php?f=KDE4/v2/moses/af-ms.txt.zip
> ```

Hi @yongzx, it seems that opus people did a major overhaul to their website. I have updated the script with the latest download link.

**Review by ljvmiranda921 — APPROVED** [review]
Hmm, the dataloader seems to work on my machine (with 53726 instances). I guess the previous were already addressed? I'm approving this now, but let's wait for @holylovenia and @yongzx 's approval / re-review.

**Comment by yongzx** [comment]
Works for me now! Approving this.

**Pull request closed without merging**
